docs/admin/install/_index.md

```diff
@@ -18,6 +18,7 @@ aliases:
   - /docs/install/knative-with-minishift
   - /docs/install/knative-with-pks
   - /docs/install/any-kubernetes-cluster
+  - /docs/install/
```

Comment:
I think we want a simple "getting started" install in addition to the admin guide; do you want to wait for that to go in and/or lift one from https://github.com/julz/docs/blob/mkdocs/docs/getting-started/getting-started.md ?

@julz FYI for possible merge issues.

Reply:
You'll laugh but I was asking Omer yesterday if it would be helpful for me to just pull all of that content into the old/main version of the website as I'm rewriting/moving things, so I'm sort of unofficially doing that to provide editing and cos it'll probably make cutting over the new site easier.
I literally just started yesterday so it's very WIP, but I'm doing it as part of a PR where I wanna also add a developer guide, cos I think it'll be easier to multitask: https://github.com/knative/docs/pull/3531/files

Reply:
Resolving since this is outside the scope of this PR